src/Compilers/Core/Portable/Compilation/NullableContext.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#nullable enable
```

Comment:
The file contains value types only.

Reply:
I know. But I'm trying to enable this for files as I touch/create them. I'll keep it on.

---
In reply to: [302581902](https://github.com/dotnet/roslyn/pull/37017#discussion_r302581902) [](ancestors = 302581902)